### ISSUE TITLE
Install database

### ISSUE THREAD

**Issue by ictinc** [juhanurmi/ahmia#36]
Hi there,

First of all thanks for this great project. I've been searching for something like this for quite a while now and it seems most of Ahmia fit my needs perfectly.

I'm trying to setup Ahmia. I find the instructions pretty hard to follow unfortunately, it seems there are some parts missing here and there.

In the readme.md it says "And after creating the SQLite database". However it doesn't say how these should be create, and what the layout needs to be. I would also like to set this up with PostgreSQL instead of SQLite, but how would I go on and do this?

If I could help out in any way with this project I would love to do so although I have minimal coding skills but tons of system/network and security administration skills.

Cheers, Ronald.

**Comment by juhanurmi** [comment]
Hi Ronald,

Yes the documentation is out of date. We are publishing completely new version of Ahmia in this month.

Would you like to join to develop Ahmia?

Cheers,
Juha

**Comment by ictinc** [comment]
Hi Juha,

Thanks for your reply. Yes, I would like to join the development of you can
point me out on where to start or what exactly you need.

Cheers,

Ronald

**Comment by juhanurmi** [comment]
Please contact me using my email address: juha dot nurmi at ahmia dot fi

And I will introduce you to our team :)

Thanks,
Juha
